server/services/interventionsService.ts

```diff
@@ -4,6 +4,7 @@ import { ApiConfig } from '../config'
 
 export interface DraftReferral {
   id: string
+  created: string
```

Comment:
Thinking out loud, I wonder if `createdAt` might be a bit clearer than plain "created" (which to me sounds like it could be a boolean value?) Ignore me if you think I'm fussing over nothing!

Reply:
yeh fine with me. i'll make the change

Reply:
Are you leaving it as `created` on the API response? If so, maybe we should just leave it as it was so they at least match up here, although I don't know how important that is.

Reply:
this already exists in the response, so would require changes on the APi too. i agree the naming does sounds like a boolean value, but it's not going to bug me too much